orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/pipeline/UpsertAsgTagsStage.groovy

```diff
@@ -27,6 +27,7 @@ import org.springframework.stereotype.Component
 
 @Component
 @CompileStatic
+@Deprecated
 class UpsertAsgTagsStage extends LinearStage {
```

Comment:
Can't `UpsertAsgTagsStage` now simply extend `UpsertServerGroupTagsStage` and `UpsertAsgTagsTask` be dropped entirely (like what we did with `UpsertAmazonLoadBalancerStage`)?